<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acMainWindow</class>
 <widget class="QMainWindow" name="acMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnCloud</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Icons/icons.qrc">
    <normaloff>:/Icons/AppIcon128.png</normaloff>:/Icons/AppIcon128.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="acMainToolbar" name="mainToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>acMainToolbar</class>
   <extends>QToolBar</extends>
   <header location="global">Components/Toolbars/acMainToolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
